### PR TITLE
feat: Issues-116 support options for WriteToTTML

### PR DESCRIPTION
**Pull request by min0625**
Issue: https://github.com/asticode/go-astisub/issues/116

### CODE REVIEW

File: ttml_test.go [hunk 1]
```diff
@@ -68,3 +68,20 @@ func TestTTMLBreakLines(t *testing.T) {
 
 	assert.Equal(t, strings.TrimSpace(string(c)), strings.TrimSpace(w.String()))
 }
+
+func TestTTMLWithOptionsNoIndent(t *testing.T) {
+	// Open
+	s, err := astisub.OpenFile("./testdata/example-in-no-indent.ttml")
```
Comment (asticode) [inline ttml_test.go:74]: Why add this new `in` file? Can't you use `./testdata/example-in.ttml` directly?

File: ttml_test.go [hunk 2]
```diff
@@ -68,3 +68,20 @@ func TestTTMLBreakLines(t *testing.T) {
 
 	assert.Equal(t, strings.TrimSpace(string(c)), strings.TrimSpace(w.String()))
 }
+
+func TestTTMLWithOptionsNoIndent(t *testing.T) {
```
Comment (asticode) [inline ttml_test.go:72]: Can you rename this test to `TestWriteToTTMLWithIndentOption`?

**Review by asticode — CHANGES_REQUESTED** [review]

File: ttml_test.go (hunk 2 above)
Comment (min0625) [reply ttml_test.go:72]: OK, will rename it later.

Comment (min0625) [reply ttml_test.go]: Fixes: [commit](https://github.com/asticode/go-astisub/pull/117/commits/83ca26d9703cf904e3f6254cecd621c7cd9fbf1f)

File: ttml_test.go (hunk 1 above)
Comment (asticode) [reply ttml_test.go:74]: I understand but I'd rather use `./testdata/example-in.ttml` here 👍

Comment (min0625) [reply ttml_test.go:74]: OK, will fix it soon.

**Comment by min0625** [comment]
Yes, I need a tag. Thank you!

**Comment by asticode** [comment]
I've created the `v0.28.0` tag 👍

**Pull request merged**